common/src/main/java/com/redhat/cloud/notifications/config/FeatureFlipper.java

```diff
@@ -183,6 +187,14 @@ public void setComplianceEmailTemplatesV2Enabled(boolean complianceEmailTemplate
         this.complianceEmailTemplatesV2Enabled = complianceEmailTemplatesV2Enabled;
     }
 
+    public boolean isIntegrationsEmailTemplatesV2Enabled() {
+        return integrationsEmailTemplatesV2Enabled;
+    }
+
+    public void setIntegrationsEmailTemplatesV2Enabled(boolean integrationsEmailTemplatesV2Enabled) {
+        this.integrationsEmailTemplatesV2Enabled = integrationsEmailTemplatesV2Enabled;
```

Comment:
Same as the other PR, lets prevent to update this on a regular run.
```suggestion
        checkTestLaunchMode();
        this.integrationsEmailTemplatesV2Enabled = integrationsEmailTemplatesV2Enabled;
```